presto-main/src/main/resources/webapp/index.html

```diff
@@ -128,6 +129,7 @@
                           queryInfo.session.source,
                           queryInfo.session.user,
                           queryInfo.state,
+                          queryInfo.errorType != null ? queryInfo.errorType : "",
```

Comment:
should the same change be propagated to query.html as query.html also has errorType field and it should be consistent with index.html. Also their is no shared .js file where we can put this common functionality so the code has to be duplicated in query.html